xml/vt_guestfs.xml

```diff
@@ -429,6 +429,38 @@ virt-ls -a sles.qcow2 /tmp/
 .X11-unix
 data.tar</screen>
    </sect3>
+   <sect3 xml:id="sec.guestfs.tools.others.log">
+    <title><command>virt-log</command></title>
+    <para>
+     <command>virt-log</command> shows the log files of the virtual machine
+     disk image or named libvirt domain. If the package
```

Comment:
"virtual machine disk image" was meant to be one single block for me (for the -a option).
virt-log is in the guestfs-tools, so nothing more to install to get it.